roles/container-engine/cri-dockerd/tasks/main.yml

```diff
@@ -21,5 +21,13 @@
   with_items:
     - cri-dockerd.service
     - cri-dockerd.socket
+  register: cri_dockerd_files
   notify:
     - restart cri-dockerd
+
+- name: Reload systemd and restart containerd   # noqa no-handler
```

Comment:
Shouldn't this actually restart the `docker` service which in turn should trigger the `containerd` restart ? In this situation `cri-dockerd` talks to docker through the docker socket not to containerd directly, that is kind of the point of maintaining this compatibility layer, we should keep the same abstraction in our code.

Reply:
Thanks, I checked restarting of docker.service on my end and got correct results. Also switched back to using handler here with flush_handlers since on my end the previous CI error didn't happen. Hope it will be the same for CI here.